packages/run-it/src/components/PerfTracker/perfUtils.ts

```diff
@@ -117,10 +139,10 @@ export class LoadTimes implements IResourceLoadTimes {
 export class PerfTimings {
   private _full = false
```

Comment:
isFull does not appear to be used. Just an observation

Reply:
Not currently. Originally I had it hooked up for enabling the `Clear` button in the perf table, but then I decided people probably want to clear whenever they want.